mlflow/server/js/src/components/ExperimentView.css

```diff
@@ -177,10 +177,6 @@ span.error-message {
   text-decoration: underline;
 }
 
-.ExperimentView .metric-param-value {
-  margin-left: 4px;
-}
-
 .ExperimentView .highlighted {
   border-radius: 5px;
   background-color: rgba(255, 242, 172, 0.5);
```

Comment:
As a nit, instead of using alpha, you should just code in the final color to make this easier to modify. Maybe the alpha was useful when these things could appear on both white and gray backgrounds, but now they're all on gray.